config/201-controller-role.yaml

```diff
@@ -67,7 +67,7 @@ rules:
     verbs: ["create"]
   - apiGroups: [""]
     resources: ["secrets"]
-    verbs: ["get", "create", "update"]
+    verbs: ["get", "create", "update", "delete"]
```

Comment:
how does adding delete fixes it ? any chance to update the commit message to explains?

Reply:
In the error log i see this message
```
secrets \"pac-gitauth-zxxz\" is forbidden: cannot set an ownerRef on a resource you can't delete: ,
```
when i searched i got this link
https://github.com/kubevirt/containerized-data-importer/pull/291

Reply:
Updated description